tests/playTests.sh

```diff
@@ -459,6 +459,19 @@ if [ -n "$DEVNULLRIGHTS" ] ; then
     ls -las $INTOVOID | grep "rw-rw-rw-"
 fi
 
+if [ -n "$READFROMBLOCKDEVICE" ] ; then
```

Comment:
nit: similar to above test,
just add a little comment here
explaining why these tests are conditional, and usage of `sudo`